src/main/java/com/dell/isg/smi/service/server/inventory/controller/ServerInventoryController.java

```diff
@@ -59,7 +59,7 @@
 
 
     @RequestMapping(value = "/hardware", method = RequestMethod.POST, headers = "Accept=application/json", consumes = "application/json", produces = "application/json")
-    @ApiOperation(value = "/hardware", nickname = "hardware", notes = "This operation allow user to get complete server hardware inventory using WSMan protocol.", response = ServerHardwareInventory.class)
+    @ApiOperation(value = "/hardware", nickname = "hardware", notes = "This operation allow user to retrieve complete server hardware inventory via the iDRAC using WSMan protocol.", response = ServerHardwareInventory.class)
```

Comment:
This operation allow**s a** user to retrieve complete server hardware inventory via the iDRAC using **the** WSMan protocol.